locales/id/id.json

```diff
@@ -171,9 +171,9 @@
     "E-Mail Address": "Alamat Surel",
     "Ecuador": "Ekuador",
     "Edit": "Edit",
```

Comment:
I just realized even though both "edit" and "sunting" are valid in Indonesian, I guess "sunting" is more appropriate here